src/DBAL/QueryFunction.php

```diff
@@ -0,0 +1,357 @@
+        $exp = 'COUNT(';
+        if ($distinct) {
+            $exp .= 'DISTINCT ';
+        }
+        $exp .= $expression . ')';
+        return new QueryExpression($exp, $alias);
+    }
+
+    /**
+     * Build a CAST SQL function call
+     * @param string|QueryExpression $expression Expression to cast
+     * @param string $type Type to cast to
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function cast(string|QueryExpression $expression, string $type, ?string $alias = null): QueryExpression
+    {
+        global $DB;
+
+        $expression = $expression instanceof QueryExpression ? $expression : $DB::quoteName($expression);
+        return new QueryExpression('CAST(' . $expression . ' AS ' . $type . ')', $alias);
+    }
+
+    /**
+     * Build a CONVERT SQL function call
+     * @param string|QueryExpression $expression Expression to convert
+     * @param string $transcoding Transcoding to use
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function convert(string|QueryExpression $expression, string $transcoding, ?string $alias = null): QueryExpression
+    {
+        global $DB;
+        $expression = $expression instanceof QueryExpression ? $expression : $DB::quoteName($expression);
+        return new QueryExpression('CONVERT(' . $expression . ' USING ' . $transcoding . ')', $alias);
+    }
+
+    /**
+     * Build a NOW SQL function call
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function now(?string $alias = null): QueryExpression
+    {
+        return new QueryExpression('NOW()', $alias);
+    }
+
+    /**
+     * Build a REPLACE SQL function call
+     * @param string|QueryExpression $expression Expression to search in
+     * @param string|QueryExpression $search String to search
+     * @param string|QueryExpression $replace String to replace
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function replace(string|QueryExpression $expression, string|QueryExpression $search, string|QueryExpression $replace, ?string $alias = null): QueryExpression
+    {
+        return self::getExpression('REPLACE', [$expression, $search, $replace], $alias);
+    }
+
+    /**
+     * Build a FROM_UNIXTIME SQL function call
+     * @param string|QueryExpression $expression Expression to convert
+     * @param string|QueryExpression|null $format Function result alias
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function fromUnixTimestamp(string|QueryExpression $expression, string|QueryExpression $format = null, ?string $alias = null): QueryExpression
```

Comment:
It should reflect the exact name of the MySQL method.
```suggestion
    public static function fromUnixtime(string|QueryExpression $expression, string|QueryExpression $format = null, ?string $alias = null): QueryExpression
```